Analysis/config/2017/sources.cfg

```diff
@@ -12,7 +12,7 @@ categories: 2j 2j0bR_noVBF 2j1bR_noVBF 2j2b+R_noVBF 2j2Lb+B_noVBF 4j1b+_VBF
 #categories: 2j 2j0bR_noVBF 2j1bR_noVBF 2j2b+R_noVBF 2j2Lb+B_noVBF 4j1b+_VBF 2j1b+B 2j1b+B_noVBF 2j0b 2j1b 2j2b+ 2j0Lb 2j1Lb 2j2Lb 2j1bR 2j2b+R
 #sub_categories: NoCuts
 sub_categories: NoCuts mh
-regions: OS_Isolated OS_AntiIsolated SS_Isolated SS_AntiIsolated SS_LooseIsolated
+regions_str: OS_Isolated OS_AntiIsolated SS_Isolated SS_AntiIsolated SS_LooseIsolated
```

Comment:
don't change how the param is called in the config. keep `regions`.  Same for 2016&2018 configs.